### PR TITLE
Android remote message: [Experiment] Increase retention through AppTP promotions

### PR DESCRIPTION
**Pull request by anikiki**
Task/Issue URL: https://app.asana.com/0/0/1204241239463178/f

### Description
Added 3 remote messages to promote app tracking protection. This is part of an experiment. See https://app.asana.com/0/72649045549333/1204249659153547/f

### Steps to test this PR

_Test 1_ Variant A from design (`zh`)
- [ ] Checkout `feature/ana/implementation_for_experiment_increase_retention_through_apptp_promotions_variants` branch from the Android repo.
- [ ] Go to `RemoteMessagingService` and replace endpoint with https://jsonblob.com/api/jsonBlob/1093199687577780224.
- [ ] Go to VariantManager and enable only variant `zh`.
- [ ] Fresh install.
- [ ] Skip onboarding.
- [ ] Ensure the expected remote message (the one with id `apptp_retention_variant_a`) appears on the new tab screen.

_Test 2_ Variant B from design (`zi`)
- [ ] Checkout `feature/ana/implementation_for_experiment_increase_retention_through_apptp_promotions_variants` branch from the Android repo.
- [ ] Go to `RemoteMessagingService` and replace endpoint with https://jsonblob.com/api/jsonBlob/1093199687577780224.
- [ ] Go to VariantManager and enable only variant `zi`.
- [ ] Fresh install.
- [ ] Skip onboarding.
- [ ] Ensure the expected remote message (the one with id `apptp_retention_variant_b`) appears on the new tab screen.

_Test 3_ Variant C from design (`zl`)
- [ ] Checkout `feature/ana/implementation_for_experiment_increase_retention_through_apptp_promotions_variants` branch from the Android repo.
- [ ] Go to `RemoteMessagingService` and replace endpoint with https://jsonblob.com/api/jsonBlob/1093199687577780224.
- [ ] Go to VariantManager and enable only variant `zl`.
- [ ] Fresh install.
- [ ] Skip onboarding.
- [ ] Ensure the expected remote message (the one with id `apptp_retention_variant_c`) appears on the new tab screen.

### NO UI changes

### CODE REVIEW

File: live/android-config/android-config.json [hunk 1]
```diff
@@ -1,5 +1,586 @@
 {
   "version": 2,
```
Comment (cmonfortep) [inline live/android-config/android-config.json:2]: before publishing, ensure you put here version 3.

Comment (anikiki) [reply live/android-config/android-config.json:2]: Thanks! I was thinking if it should be 2 or 3 :)

File: live/android-config/android-config.json [hunk 2]
```diff
@@ -1,5 +1,586 @@
...
+      "matchingRules": [
+        3
+      ]
+    }
+  ],
+  "rules": [
+    {
+      "id": 1,
+      "attributes": {
+        "locale": {
```
Comment (cmonfortep) [inline live/android-config/android-config.json:426]: Not sure we need a locale rule. Let's discuss that offline.

Comment (anikiki) [reply live/android-config/android-config.json]: Updated, thanks!

**Pull request merged**